File: robin/python/level1/study5/sum_between_two_nums.py
def solution(a, b):
    max_num = max(a, b) + 1
    min_num = min(a, b)

    return sum(list(range(min_num, max_num)))


if __name__ == '__main__':
    a = 3
    b = 5
    print(solution(a, b))

File: robin/java/src/programmers/level1/study5/SumBetweenTwoNums.java
package programmers.level1.study5;

import java.util.stream.LongStream;

public class SumBetweenTwoNums {

    public static long solution(int a, int b) {

        return LongStream.range(Math.min(a, b), Math.max(a, b) + 1).sum();
    }


    public static void main(String[] args) {
        int a = 3;
        int b = 5;

        System.out.println(solution(a, b));
    }
}
